doc/man5/flux-config-policy.rst

```diff
@@ -0,0 +1,108 @@
+=====================
+flux-config-policy(5)
+=====================
+
+
+DESCRIPTION
+===========
+
+The ``policy`` table captures general site preferences for job request defaults
+and limits, as described in RFC 33.
+
+Each queue defined in the ``queues`` table described in
+:man5:`flux-config-queues` may have a ``policy`` sub-table that follows the
+same rules as the main table.  The per-queue table overrides the general table
+for jobs submitted to that queue.
+
+DEFAULTS
+========
+
+Default values for job requests may be configured in the
+``policy.jobspec.defaults.system`` table.  If a job request does not specify
+a value for a system attribute that is configured in the table, the configured
+value is substituted.  Some common examples are:
+
+policy.jobspec.defaults.system.duration (double or FSD string)
+   (optional) If a job is submitted without specifying a job duration,
+   this value is substituted.
+
+policy.jobspec.defaults.system.queue (string)
+   (optional) If a job is submitted without specifying a queue name,
+   this value is substituted.
+
+LIMITS
+======
+
+Limits may be configured in the ``policy.limits`` table.  Job requests are
+rejected at submission time if they violate configured limits.
+
+.. note::
+   It is possible to set a queue-specific limit to an `unlimited` value,
+   and thereby defeat a configured general limit for jobs submitted to that
+   queue.  The actual value used to indicate `unlimited` varies by limit
+   type and is noted below.
+
+policy.limits.duration (double or FSD string)
+   (optional) Maximum duration that a job may request (0 = unlimited).
```

Comment:
units?

Reply:
Oh yeah and "double" is not a toml or JSON type.  TOML uses "real".